src/components/todo/todo-title-checkbox.tsx

```diff
@@ -33,7 +33,17 @@ export default function TodoTitleAndCheckBox({
           id={`todo-check-${index}`}
           checked={isDone}
           aria-checked={isDone}
-          onChange={() => toggleStatus({ ...todo, isDone: !todo.isDone })}
+          onChange={() =>
+            toggleStatus({
+              todoId: todo.id,
+              newTodo: {
+                fileUrl: todo.fileUrl,
+                isDone: !todo.isDone,
+                title: todo.title,
+                linkUrl: todo.linkUrl,
```

Comment:
이 컴포넌트에서는 isDone의 상태만 바뀌는 것이니 아래처럼 수정하는 것이 더 명확할 것 같은데 어떠신가요?

```suggestion
                ...todo,
                isDone: !todo.isDone,
```

Reply:
더 좁은 범위의 타입이라 가능하군요. 이렇게 구현하도록 하겠습니다!!